pulp_rpm/app/serializers.py

```diff
@@ -526,3 +527,55 @@ class Meta:
             'module', 'stream', 'profiles'
         )
     model = ModulemdDefaults
+
+
+class DistributionTreeSerializer(SingleArtifactContentSerializer):
+    """
+    DistributionTree serializer.
+    """
+
+    header_version = serializers.CharField(
+        help_text=_("Header Version.")
+    )
+    release_name = serializers.CharField(
+        help_text=_("Release name.")
+    )
+    release_short = serializers.CharField(
+        help_text=_("Release short.")
+    )
+    release_version = serializers.CharField(
+        help_text=_("Release version.")
+    )
+    release_is_layered = serializers.BooleanField()
+
+    base_product_name = serializers.CharField(
+        help_text=_("Base Product name."), allow_null=True
+    )
+    base_product_short = serializers.CharField(
+        help_text=_("Base Product shor."), allow_null=True
```

Comment:
typo s/shor/short/

And what is it? Base Product short? Short name? please add it to help_text